tsl/src/continuous_aggs/create.c

```diff
@@ -321,7 +322,8 @@ create_bucket_function_catalog_entry(int32 matht_id, bool experimental, const ch
 	table_close(rel, RowExclusiveLock);
 }
 
-/* create hypertable for the table referred by mat_tbloid
+/*
+ * Create hypertable for the table referred by mat_tbloid
```

Comment:
There are too much formatting changes in this PR. This make the `git blame` in the future hard to figure out exactly who introduced the change. Instead you can send a separate PR for those formatting changes also adding the commit to the .git-blame-ignore-revs